ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+DEFAULT_WORKDIR="/var/lib/jenkins"
+WORKDIR=$DEFAULT_WORKDIR
+KUBECONFIG_PATH="$DEFAULT_WORKDIR/.kube/config"
+TESTCASE=""
+TEST_FAILURE=false
+MODE="report"
+DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")
+TESTBED_TYPE="static"
+GO_VERSION=$(head -n1 "${WORKSPACE}/build/images/deps/go-version")
+IMAGE_PULL_POLICY="Always"
+K8S_VERSION="1.23.6-00"
+
+CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]"
+NETWORKPOLICY_SKIP="should allow egress access to server in CIDR block|should enforce except clause while egress access to server in CIDR block"
+
+CONTROL_PLANE_NODE_ROLE="controlplane,etcd"
```

Comment:
May I know why etcd is added here?

Reply:
because the control plane node role in rancher cluster is controlplane,etcd. not only controlplane

Reply:
Got it, maybe it's better to add a comment here.

Reply:
In my kubernetes cluster, the roles are reflected on the node labels, too. For example, my control plane node has two roles "control-plane,master", so it has two labels `node-role.kubernetes.io/control-plane: ""`, `node-role.kubernetes.io/master: ""`. I can get the cp node via a label selector:

```
kubectl get nodes -l node-role.kubernetes.io/control-plane -o wide
```

Can you check whether Rancher cluster control plane node has a label, too? I think getting control plane node by label is cleaner than matching the roles in awk.